XSConsoleLangFriendlyNames.py

```diff
@@ -20,7 +20,7 @@
 class LangFriendlyNames:
   @classmethod
   def Translate(cls, inTag):
-    return cls.friendlyNamesMap.get(inTag, inTag)
+    return cls.friendlyNamesMap.get(inTag, None)
```

Comment:
This file is (supposed to be) autogenerated from `GenerateXSConsoleLangFriendlyNames.rb` so a similar change should be made there too.